packages/react-components/index.js

```diff
@@ -12,10 +13,44 @@ module.exports = (neutrino, opts = {}) => {
       title: 'React Preview'
     },
     manifest: mode === 'development',
+    babel: {},
     externals: opts.externals !== false && {},
     style: { extract: { plugin: { filename: '[name].css' } } }
   }, opts);
 
+  Object.assign(options, {
+    babel: compileLoader.merge({
```

Comment:
The react-components preset inherits from the react preset, which already configures `@babel/preset-env`. As such I don't think we need to add this here, and instead could just override the targets passed to the react preset.

Reply:
@edmorley so the entire `Object.assign` portion can be safely ignored?

Reply:
Yeah I believe so. I'd just set default values for `options.targets` and let those be passed down to the react (and thus web) presets.

Reply:
Not sure this is entirely true.

So I see that basically every package starts with an empty babel object and builds on it ->

Reply:
The reason the other presets start with an empty babel object, is that they need to customise the list of plugins used.

However for this PR, the only thing that needs changing, is the list of browser/node targets, which can be passed down via the standard options, and will cause the existing web preset functionality to use them:
https://github.com/mozilla-neutrino/neutrino-dev/blob/e597c615e2c76e9986b6262249e35001a00efc0f/packages/web/index.js#L94